specific/cmo_sale/wizard/cal_manage_fee.py

```diff
@@ -13,8 +13,14 @@ class SaleCalManageFee(models.TransientModel):
     @api.multi
     def calculate_management_fee(self):
         self.ensure_one()
-        order_line = self.env['sale.order.line'].browse(self._context['order_line_id'])
+        order_line = self.env['sale.order.line'].browse(
+            self._context['order_line_id'])
         quote = self.env['sale.order'].browse(order_line.order_id.id)
-        fee = quote.amount_before_management_fee * self.percent_rate / 100
+        if order_line.sale_layout_custom_group_id:
+            fee = quote._get_amount_by_custom_group(
```

Comment:
Try use float_round() to round the value, especially if there is division /
Note: this case, your code is ok because it is 100. But make it good habbit.